### DISCUSSION TITLE
第五节课作业(25班)

### DISCUSSION THREAD

**Discussion by zcg-joker** [InternLM/Tutorial#456] [作业]
作业链接放在这里！
笔记链接**不放在**这里！

**Comment by zengya55** [comment]
https://github.com/zengya55/InterLM_Learning/blob/main/lesson5/lesson5%20lmdepoly%20%E4%BD%9C%E4%B8%9A.md

**Comment by FableFatale** [comment]
https://www.cnblogs.com/yunyetui/p/17978567

**Comment by sxswz213** [comment]
https://blog.csdn.net/sxswz/article/details/135734158?spm=1001.2014.3001.5501

**Comment by liliang1918** [comment]
https://blog.csdn.net/lalala12ll/article/details/135832209?spm=1001.2014.3001.5502

**Comment by BaiYu96** [comment]
https://github.com/BaiYu96/internLM-learm/blob/main/homework/lesson-5.md

**Comment by ykuang** [comment]
https://github.com/ykuang/internlm-tutorial/blob/main/notes/lesson5_homework.md

**Comment by Cyberworldvoyager** [comment]
https://github.com/Cyberworldvoyager/Interlm_lecture/blob/main/lecture%205.pdf

**Comment by GuoYiFantastic** [comment]
https://github.com/GuoYiFantastic/InternLM_training_camp/blob/main/LMDeploy%20%E5%A4%A7%E6%A8%A1%E5%9E%8B%E9%87%8F%E5%8C%96%E9%83%A8%E7%BD%B2%E5%AE%9E%E8%B7%B5/%E4%BD%9C%E4%B8%9A.md

**Comment by juxingyiwan** [comment]
https://github.com/juxingyiwan/InternLM_Camp_202401/tree/main/Lecture5
